elkm1_lib/keypads.py

```diff
@@ -61,6 +69,14 @@ def _kc_handler(self, keypad: int, key: int) -> None:
             name = ""
         self.elements[keypad].setattr("last_keypress", (name, key), True)
 
+    def _kf_handler(self, keypad: int, key: str, chime_mode: list[int]) -> None:
+        self.elements[keypad].last_function_key = None  # Force a change notification
```

Comment:
Since we only care about "real" function key presses, setting this to `FunctionKeys.FORCE_KF_SYNC` is appropriate. This should address the typing error.

Reply:
done